src/requireConfig.spec.ts

```diff
@@ -14,7 +14,6 @@ const options: Options = {
   config: new Config({}),
   filteredTargets: [],
   force: false,
-  json: false,
   nonInteractive: false,
   interactive: false,
```

Comment:
Most of these mock options have both `nonInteractive` and `interactive` set to false. This is clearly a mistake. Lmk if you'd like me to fix it in a follow-up PR.

Reply:
Go for it!